broker/data-access-layer/iaas/test/iaas.AwsClient.spec.js

```diff
@@ -0,0 +1,293 @@
+'use strict';
+
+const { AwsClient } = require('@sf/iaas');
+
+describe('iaas', function () {
+  describe('AwsClient', function () {
+    this.slow(200);
+
+    describe('#createDiskFromSnapshot', function () {
```

Comment:
Took this entirely from `iaas.ClouProviderClient.js` except changed the `class` used to create the client.